.github/workflows/kind.yaml

```diff
@@ -26,11 +26,12 @@ jobs:
           make build
           IMG=flotta-operator:latest make docker-build
           kind load docker-image flotta-operator:latest
-          timeout 1m make deploy IMG=flotta-operator || retVal=$?
+          timeout 1m make deploy IMG=flotta-operator TARGET=kind || retVal=$?
```

Comment:
@machacekondra do you have any idea how come the created file operator.yaml cannot be found? 
```
...
kubectl apply -f /tmp/tmp.SMmh0PSDvt/flotta-operator.yaml
error: the path "/tmp/tmp.SMmh0PSDvt/flotta-operator.yaml" does not exist
make: *** [Makefile:145: deploy] Error 1
Make run failed
Error: Process completed with exit code 1.
```
from https://github.com/project-flotta/flotta-operator/runs/5189563062?check_suite_focus=true#step:4:985

Reply:
fixed